browser/help/topics/v3-ht-annotations.md

```diff
@@ -165,22 +165,22 @@ The gnomAD v3 Hail Table annotations are defined below:
   - **bin_freq**: Bin frequencies for the age histogram. This is the number of records found in each bin.
   - **n_smaller**: Count of age values falling below lowest histogram bin edge.
   - **n_larger**: Count of age values falling above highest histogram bin edge.
-- **in_silico_predictors**: Variant prediction annotations. We put together the prediction scores from multiple in silico predictors for variants that are predicted to be missense, impacting protein function, evolutionarily conserved or splice-altering. We focus on MANE Select and canonical transcripts if the prediction scores are available for multiple transcripts.
+- **in_silico_predictors**: Variant prediction annotations. Struct contains prediction scores from multiple in silico predictors for variants that are predicted to be missense, impacting protein function, evolutionarily conserved, or splice-altering. We chose scores for either MANE Select or canonical transcripts if a prediction score was available for multiple transcripts.
   - **revel**: Rare Exome Variant Ensemble Learner (REVEL) [score](https://sites.google.com/site/revelgenomics/about). An ensemble score for predicting the pathogenicity of missense variants (based on 13 other variant predictors).
-    - **revel_max**: Score predicting the pathogenicity of missense variants with a range of 0 to 1. Variants with higher scores are predicted to be more likely to be deleterious. We prioritize max scores for MANE Select transcripts where possible and otherwise report a score for the canonical transcript.
+    - **revel**: Rare Exome Variant Ensemble Learner (REVEL) [score](https://sites.google.com/site/revelgenomics/about). An ensemble score for predicting the pathogenicity of missense variants (based on 13 other variant predictors).
```

Comment:
wait, it looks like something went awry here. the line with `revel_max` no longer exists here?